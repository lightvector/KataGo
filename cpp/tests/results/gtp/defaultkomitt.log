: GTP Engine starting...
: KataGo v1.8.2
: Using koPOSITIONALscoreAREAtaxNONEsui0 rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: hasButton = false
: Config override: koRule = POSITIONAL
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/defaultkomitt.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: multiStoneSuicideLegal = false
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rules = 
: Config override: scoringRule = AREA
: Config override: searchRandSeed = forTesting
: Config override: taxRule = NONE
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . @ . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0komi7.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 96
NN batches: 96
NN avg batch size: 1
PV: Q17 R4 D3 D17
Tree:
: T   0.01c W  -0.90c S   0.91c ( -0.0 L  +0.0) N     100  --  Q17 R4 D3 D17
---Black(^)---
Q17 : T   0.96c W  -0.04c S   0.99c ( +0.2 L  +0.3) LCB   -4.97c P  8.89% WF  11.1 PSV      20 N      11  --  Q17 R4 D3 D17
Q3  : T   0.54c W  -0.36c S   0.90c ( +0.1 L  +0.2) LCB   -6.69c P  9.01% WF  10.0 PSV      10 N      10  --  Q3 R16 D3 D17
C4  : T   0.69c W  -0.65c S   1.34c ( -0.1 L  -0.0) LCB  -11.97c P  8.74% WF  10.0 PSV      10 N      10  --  C4 R16 C16
D3  : T   1.14c W  -0.12c S   1.26c ( +0.2 L  +0.3) LCB   -9.58c P  8.47% WF  10.1 PSV      10 N      10  --  D3 R16 C16
C16 : T   0.85c W  -0.50c S   1.35c ( +0.0 L  +0.1) LCB  -10.30c P  8.31% WF  10.1 PSV      10 N      10  --  C16 D3 R16
R4  : T  -0.92c W  -1.54c S   0.63c ( -0.2 L  +0.0) LCB  -11.37c P  8.99% WF   8.9 PSV       9 N       9  --  R4 R16 D3 D17
D17 : T  -0.01c W  -1.20c S   1.18c ( -0.1 L  -0.1) LCB  -16.11c P  8.04% WF   8.0 PSV       8 N       8  --  D17 D3 R16
R16 : T   0.37c W  -0.64c S   1.01c ( +0.1 L  +0.1) LCB  -13.42c P  7.48% WF   8.0 PSV       8 N       8  --  R16 D3 C16
Q4  : T  -1.00c W  -1.53c S   0.52c ( -0.2 L  -0.1) LCB  -26.45c P  5.64% WF   5.0 PSV       5 N       5  --  Q4 D3 D17
D4  : T  -1.12c W  -1.62c S   0.50c ( -0.2 L  -0.0) LCB  -92.12c P  4.46% WF   4.0 PSV       4 N       4  --  D4 R4 D17

: = R4
: GPU -1 finishing, processed 96 rows 96 batches
: All cleaned up, quitting
