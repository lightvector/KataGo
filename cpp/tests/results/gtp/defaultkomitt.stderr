KataGo v1.8.0
Using koPOSITIONALscoreAREAtaxNONEsui0 rules initially, unless GTP/GUI overrides this
Loaded config configs/gtp_example.cfg and/or command-line and query overrides
Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
Model name: g170-b6c96-s175395328-d26788732
GTP ready, beginning main protocol loop
MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . @ . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0komi7.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 96
NN batches: 96
NN avg batch size: 1
PV: Q17 R4 D3 D17
Tree:
: T   0.01c W  -0.90c S   0.91c ( -0.0 L  +0.0) N     100  --  Q17 R4 D3 D17
---Black(^)---
Q17 : T   0.96c W  -0.04c S   0.99c ( +0.2 L  +0.3) LCB   -4.97c P  8.89% WF  6.80% PSV      20 N      11  --  Q17 R4 D3 D17
Q3  : T   0.54c W  -0.36c S   0.90c ( +0.1 L  +0.2) LCB   -6.69c P  9.01% WF  6.75% PSV      10 N      10  --  Q3 R16 D3 D17
C4  : T   0.69c W  -0.65c S   1.34c ( -0.1 L  -0.0) LCB  -11.97c P  8.74% WF  6.77% PSV      10 N      10  --  C4 R16 C16
D3  : T   1.14c W  -0.12c S   1.26c ( +0.2 L  +0.3) LCB   -9.58c P  8.47% WF  6.82% PSV      10 N      10  --  D3 R16 C16
C16 : T   0.85c W  -0.50c S   1.35c ( +0.0 L  +0.1) LCB  -10.30c P  8.31% WF  6.79% PSV      10 N      10  --  C16 D3 R16
R4  : T  -0.92c W  -1.54c S   0.63c ( -0.2 L  +0.0) LCB  -11.37c P  8.99% WF  6.60% PSV       9 N       9  --  R4 R16 D3 D17
D17 : T  -0.01c W  -1.20c S   1.18c ( -0.1 L  -0.1) LCB  -16.11c P  8.04% WF  6.69% PSV       8 N       8  --  D17 D3 R16
R16 : T   0.37c W  -0.64c S   1.01c ( +0.1 L  +0.1) LCB  -13.42c P  7.48% WF  6.73% PSV       8 N       8  --  R16 D3 C16
Q4  : T  -1.00c W  -1.53c S   0.52c ( -0.2 L  -0.1) LCB  -26.45c P  5.64% WF  6.60% PSV       5 N       5  --  Q4 D3 D17
D4  : T  -1.12c W  -1.62c S   0.50c ( -0.2 L  -0.0) LCB  -92.12c P  4.46% WF  6.60% PSV       4 N       4  --  D4 R4 D17
